pop-0001.txt

```diff
@@ -0,0 +1,623 @@
+POP: 1
```

Comment:
I know that PEPs are actual `.txt`s, and they're written in reST.  But I propose we actually maintain these as reST files (e.g. rename this as `pop-0001.rst`) for being able to easily process in Sphinx (with eventual publishing in the kit at kit.pyladies.com) as well as auto-syntax highlighting for IDEs/text editors.